FastSimulation/Tracking/python/IterativePixelLessStep_cff.py

```diff
@@ -33,6 +33,8 @@
 #                                     'TEC1_pos+TEC2_pos','TEC2_pos+TEC3_pos','TEC3_pos+TEC4_pos','TEC3_pos+TEC5_pos','TEC4_pos+TEC5_pos',
 #                                     'TID1_neg+TID2_neg','TID2_neg+TID3_neg',
 #                                     'TEC1_neg+TEC2_neg','TEC2_neg+TEC3_neg','TEC3_neg+TEC4_neg','TEC3_neg+TEC5_neg','TEC4_neg+TEC5_neg']
+
+from RecoLocalTracker.SiStripClusterizer.SiStripClusterChargeCut_cfi import *
```

Comment:
Are you sure this is needed"
We only use the layer list from the seedlayer, 
and copy it to the cfg of the fast trajectory seed producer.
The module itself is not run.

May be there is something I don't understand about the python processing...